client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -375,7 +382,7 @@ const EmailMailboxActionMenu = ( { account, domain, mailbox } ) => {
 					visible={ removeTitanMailboxDialogVisible }
 				/>
 			) }
-			<EllipsisMenu position="bottom" className="email-mailbox-action-menu__main">
+			<EllipsisMenu position="bottom left" className="email-mailbox-action-menu__main">
```

Comment:
This is a styling fix that prevents overflow trouble with the action menu on smaller screens (e.g. laptops). Using `bottom left` matches the behavior in the DNS editor. Here are two screenshots that illustrate the difference this makes:

<img width="1442" alt="before" src="https://user-images.githubusercontent.com/1101677/171825667-9317ee27-f7e3-45a3-8d64-9a7d02e58b9c.png">

<img width="1442" alt="after" src="https://user-images.githubusercontent.com/1101677/171825689-a36f3180-0506-4466-bfe6-6776470718c4.png">